src/modules/product/repositories/productRepositories.ts

```diff
@@ -142,6 +142,12 @@ const sellerGetProductById = async (shopId: string,productId: string) => {
   });
 };
 
+const findProductToWishListById = async (productId:string) => {

```

Comment:
Why here you find by productId only? 
I think better you can find by productId and userId to avoid getting wishList of other users.